[*] Running ...
[*] Dataset loaded: /home/dokhanhnam1199/QD/problems/cvrp_aco/dataset/train50_dataset.npy with 10 instances.
Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/cvrp_aco/eval.py", line 60, in <module>
    obj = solve(node_pos, demand)
  File "/home/dokhanhnam1199/QD/problems/cvrp_aco/eval.py", line 28, in solve
    heu = heuristics(dist_mat.copy(), node_pos.copy(), demand.copy(), CAPACITY) + 1e-9
  File "/home/dokhanhnam1199/QD/problems/cvrp_aco/gpt.py", line 12, in heuristics_v2
    # Initialize routes randomly
NameError: name 'KMeans' is not defined
37
9
144.75398259382442
